.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,21 @@
+name: Build JoSS paper
+
+on: [push]
```

Maybe not on every push, on every changes on the folder could do the job ?

```suggestion
on:
  workflow_dispatch:
  push:
    paths:
      - 'docs/paper/**'
```